config/file/parse.go

```diff
@@ -109,11 +111,8 @@ func parseRawConfig(raw unmarshaledConfig) (config.Config, error) { //nolint:goc
 		fields = append(fields, config.FieldGlobalTimeout)
 	}
 
-	body, err := config.GetBodyContent(raw.Request.Body)
-	if err != nil {
-		fmt.Println(err)
-	}
-	if body != "" {
+	body := config.Body{Type: raw.Request.Body.Type, Content: []byte(raw.Request.Body.Content)}
+	if !reflect.DeepEqual(body, config.NewBody("", "")) {
```

Comment:
`reflect` is heavy, here we're checkin if the type is not empty string and the content is not empty slice.

```suggestion
	if body.Type != "" && len(body.Content) > 0 {
```